scala3doc/src/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -123,6 +123,26 @@ trait ClassLikeSupport:
           .map { _ =>
             parseMethod(dd.symbol, kind = Kind.Given(getGivenInstance(dd).map(_.asSignature), None))
           }
+          
+      case dd: DefDef if !dd.symbol.isHiddenByVisibility && dd.symbol.isExported =>
+        val exportedTarget = dd.rhs.flatMap { 
```

Comment:
```
dd.rhs.collect { 
    case a: Apply => a.fun
    case s: Select => s
}
```
will do the same what `flatmap` and `map`

Reply:
Will change